packages/web-components/src/components/va-button-pair/va-button-pair.scss

```diff
@@ -0,0 +1,65 @@
+@forward 'settings';
+
+@use 'usa-button-group/src/styles/usa-button-group';
+
+@import '../../global/formation_overrides';
+
+:host {
+  display: flex;
+  flex-direction: column;
+}
+
+va-button::part(button) {
+  margin-right: 0;
+}
```

Comment:
I have added this margin reset because `.usa-button` adds right margin that makes the spacing between two va-button's too large:

<img width="296" alt="Screenshot 2023-03-24 at 2 21 13 PM" src="https://user-images.githubusercontent.com/872479/227621158-513dd3ae-78b7-4ec4-ae0e-e69be552939b.png">

**Before**

<img width="294" alt="Screenshot 2023-03-24 at 2 21 31 PM" src="https://user-images.githubusercontent.com/872479/227621118-c79dc7a6-0f5a-4447-96e3-2177f65e5996.png">

**After**
<img width="332" alt="Screenshot 2023-03-24 at 2 21 22 PM" src="https://user-images.githubusercontent.com/872479/227621394-52d6d792-5843-4030-910b-6d40cfd9a65e.png">